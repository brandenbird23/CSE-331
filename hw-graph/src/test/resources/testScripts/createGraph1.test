# Create Graph Test File
# Test to create a graph
CreateGraph graph1

# Print the nodes in the graph
ListNodes graph1