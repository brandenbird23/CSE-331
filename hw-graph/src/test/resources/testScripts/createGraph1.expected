# Create Graph Expected File
# Test to create a graph
created graph graph1

# Print the nodes in the graph
graph1 contains: